common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
```

Comment:
I think archmagi size should be like 50-25 since it's a very high and prestigious title.

Reply:
👌

Reply:
> What is a reason why you have united naga and night elven units, but high elven ones are separate?

Didn't we say that spellbreakers are only after the fall of silvermoon/3rd war? at least they would have to be split from the others.

Reply:
> Didn't we say that spellbreakers are only after the fall of silvermoon/3rd war? at least they would have to be split from the others.

What about farstriders and dragonhawks?

Reply:
I'm not THAT into the lore when they both became part of the army, but I guess we could combine them.

Reply:
1. What will be the verdict for the tech?
2. Fixed high elves having generic maa
3. From the comments I got, I made it so that humans and most dwarfs are allowed to unlock it, but in history, I only gave it to them on the last bookmark 605, because they didn't have it before. I think the player could decide if they want it earlier, no?

Reply:
> What will be the verdict for the high elf/blood elf tech?

Combine them?

> From the comments I got, I made it so that humans and most dwarfs are allowed to unlock it, but in history, I only gave it to them on the last bookmark 605, because they didn't have it before. I think the player could decide if they want it earlier, no?

Fair.

Reply:
Combine spellbreakers together with the others? Last discussion you said that spellbreakers aren't a thing before the fall of silvermoon...